sources/osgShadow/ShadowMapAtlas.js

```diff
@@ -20,6 +22,7 @@ var ShadowMapAtlas = function(settings) {
     this._lights = [];
     this._shadowMaps = [];
     this._viewportDimension = [];
+    this._scissors = [];
```

Comment:
you probably dont need this variable anymore maybe you can just reference the camera